lib/fade-image/test/index.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import FadeImage from '../';
+
+const props = {
+  isBackground: true
+};
+
+describe('Component', function () {
+  it('should render our <FadeImage /> component', function (done) {
+    const wrapper = shallow(<FadeImage {...props} />);
+    const children = wrapper.children().nodes;
+    expect(children).to.have.length(0);
```

Comment:
It's not obvious how this assertion relates to the functionality under test - namely `it should render our <FadeImage /> component`